doc-src/dev.rst

```diff
@@ -431,8 +435,8 @@ Item 2 is tested; i.e., if you break the RPM build, the test suite will fail.
 This section describes how to build the RPMs and the pain we've hopefully
 abstracted away.
 
-Prerequisites
--------------
+Dependencies
+------------
 
   * Python 2.7
```

Comment:
It would be nice if we had a homogeneous python version requirement. Could we list python3 as a standard requirement with the note "should work with python2"?

Reply:
I agree.